From 8b734a0dea7d48b27ea7f21ac2bef4c336fe303d Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Thu, 8 Mar 2018 17:41:33 +1000
Subject: [PATCH] hwdb: add Dell Inspiron 3537 axis overrides (#8388)

PS/2 and RMI4 entries

https://bugs.freedesktop.org/show_bug.cgi?id=105370
(cherry picked from commit 2b5f20a99d4e94be90ee0b2cfa602b3ed3c1ea94)
---
 hwdb/60-evdev.hwdb | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/hwdb/60-evdev.hwdb b/hwdb/60-evdev.hwdb
index 2c6fdb1b69..718befaaba 100644
--- a/hwdb/60-evdev.hwdb
+++ b/hwdb/60-evdev.hwdb
@@ -144,6 +144,20 @@ evdev:name:AlpsPS/2 ALPS GlidePoint*:dmi:bvn*:bvr*:bd*:svnDellInc.:pnVostro1510*
  EVDEV_ABS_00=::14
  EVDEV_ABS_01=::18
 
+# Dell Inspiron 3537 - PS/2
+evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnDellInc.:pnInspiron3537*
+ EVDEV_ABS_00=1268:5675:41
+ EVDEV_ABS_01=1101:4792:61
+ EVDEV_ABS_35=1268:5675:41
+ EVDEV_ABS_36=1101:4792:61
+
+# Dell Inspiron 3537 - RMI4
+evdev:name:Synaptics TM2382-001:dmi:*svnDellInc.:pnInspiron3537*
+ EVDEV_ABS_00=::24
+ EVDEV_ABS_01=::34
+ EVDEV_ABS_35=::24
+ EVDEV_ABS_36=::34
+
 # Dell Inspiron N5040
 evdev:name:AlpsPS/2 ALPS DualPoint TouchPad:dmi:bvn*:bvr*:bd*:svnDellInc.:pnInspironN5040*
  EVDEV_ABS_00=25:2000:22
-- 
2.17.0

